languages/swift/exercises/concept/loops/.docs/instructions.md

```diff
@@ -0,0 +1,55 @@
+You are an elf working in Santa Claus' logistics division and you have been given a pair of tasks from the boss to help determine the gifts that Santa will deliver.
+
+It is believed that Santa sorts the various children of the world into naughty and nice to determine who gets presents, but that's not the case. What Santa really does is rank the children on a scale from 0 on up, where the higher the rank, the better the present the child gets. This ranking is based on an arcane formula that only Santa fully understands. But when it comes to the implementation of the system you know this much.
+
+- Each year, Santa distills the ranking into a "magic" hash function.
+- Every child in the world is represented in Santa's privacy-preserving database as an integer ID number in the range [1, 4,000,000,000).
+- The ranking level of each child can be encoded by using the magic hash to transform the ID into a hashed ID.
+- The ranking level can be extracted by computing the Harshad level of the hashed ID.
+
+## Tasks
+
+## 1. Hash the IDs from the database
+
+Your first task is to write a function that given an array of ID numbers, takes each ID, hashes it with Santa's magic hash, and constructs a tuple of the original ID and the hashed ID, creating a new array of these tuples.
+
+Santa's magic hash is implemented by multiplying the ID by the magic constant _p_ and taking the remainder modulo another magic constant, _m_. This year, you are given the following magic constants:
+
+- m = `3_999_999_979`
+- p = `2_305_843_009`
+
+```swift
+let ids = [980035519, 1402277754, 1821223877]
+hashIDs(ids)
+// => [(980035519, 1921828986), (1402277754, 712641739), (1821223877, 2651471804)]
+```
+
+## 2. Compute the ranking level for a hashed ID
```

Comment:
Yeay, that sounds little challenging to solve. :)
I find the explanation with the 2 given examples understandable.
I'm little bit worried, that given hints in `hints.md` might not be sufficient for students, who want hints. What about giving more guidance by describing the rough implementation idea: 1) a loop checking for the "is digital sum a multiple or number?" constraint (with more explanation, when to exit), which is using 2) an inner loop, which calculates the digital sum of a given number by iterating the digits.